express/properties/non_scalar/averaged_potential.py

```diff
@@ -0,0 +1,36 @@
+import numpy as np
+from scipy.signal import find_peaks
+from express.parsers.settings import Constant
+from express.properties.non_scalar import NonScalarProperty
+
+
+class AveragedPotential(NonScalarProperty):
+    """
+    Holds planar and macroscopic average of a potential (e.g. electrostatic potential).
+    Additionally, minima are included as they can be of further use (e.g. potential lineup method).
+    """
+
+    def __init__(self, name, parser, *args, **kwargs):
+        super(AveragedPotential, self).__init__(name, parser, *args, **kwargs)
+
+        self.data = self.safely_invoke_parser_method("averaged_quantity")
```

Comment:
I think our default units (defined inside property classes) for length is angstrom throughout express elsewhere. The conversion, thus should happen in the app-specific parsers.